src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubectlBuildWrapper.java

```diff
@@ -113,8 +120,14 @@ public void setUp(Context context, Run<?, ?> build, FilePath workspace, Launcher
             tlsConfig = " --insecure-skip-tls-verify=true";
         }
 
+        if(this.https_proxy != null  && !https_proxy.isEmpty()) {
+            kubectlBegin += "HTTPS_PROXY="+this.https_proxy+" kubectl config --kubeconfig=\"";
+        } else {
+            kubectlBegin += "kubectl config --kubeconfig=\"";
+        }
+
         int status = launcher.launch()
-                .cmdAsSingleString("kubectl config --kubeconfig=\"" + configFile.getRemote()
+                .cmdAsSingleString(kubectlBegin + configFile.getRemote()
```

Comment:
Use `launcher.launch().envs("HTTPS_PROXY="+this.https_proxy).cmdAsSingleString...` (possibly refactor it to a method to avoid repeating it)

Reply:
I'll see if this way works in my environment and make the change.